boards/arm/stm32h7/nucleo-h743zi/configs/netnsh/defconfig

```diff
@@ -85,3 +86,4 @@ CONFIG_USART3_SERIAL_CONSOLE=y
 CONFIG_USBHOST=y
 CONFIG_USBHOST_MSC=y
 CONFIG_USBHOST_MSC_NOTIFIER=y
+CONFIG_USER_ENTRYPOINT="nsh_main"
```

Comment:
remove, the option change to CONFIG_INIT_ENTRYPOINT at line 30

Reply:
solved